src/commonMain/kotlin/io/github/koalaplot/core/pie/PieChart.kt

```diff
@@ -367,8 +437,12 @@ public fun PieChart(
     minPieDiameter: Dp = 100.dp,
     maxPieDiameter: Dp = 300.dp,
     forceCenteredPie: Boolean = false,
-    pieAnimationSpec: AnimationSpec<Float> = KoalaPlotTheme.animationSpec,
-    labelAnimationSpec: AnimationSpec<Float> = tween(LabelFadeInDuration, 0, LinearOutSlowInEasing),
+    startAnimationUseCase: StartAnimationUseCase =
+        StartAnimationUseCase(
+            executionType = StartAnimationUseCase.ExecutionType.Default,
+            KoalaPlotTheme.animationSpec,
+            tween(LabelFadeInDuration, 0, LinearOutSlowInEasing)
+        ),
```

Comment:
The original version of this PieChart overload (which doesn't take a LabelPositionProvider) should also be preserved for API compatibility.

Reply:
Done.